xbmc/pvr/epg/EpgContainer.cpp

```diff
@@ -245,6 +250,24 @@ void CPVREpgContainer::Stop(void)
   m_bStarted = false;
 }
 
+void CPVREpgContainer::OnPVRRecordingsEvent(const PVRRecordingsEvent& event)
+{
+  if (event.recording->BroadcastUid() == EPG_TAG_INVALID_UID )
+    return;
+
+  const std::shared_ptr<CPVRChannel> channel
+    = CServiceBroker::GetPVRManager().ChannelGroups()->GetByUniqueID(event.recording->ChannelUid(),
+                                                                     event.recording->ClientID());
+  const std::shared_ptr<CPVREpgInfoTag> epgTag = GetTagById(channel, event.recording->BroadcastUid());
+  if (!epgTag)
+    return;
+
+  if (event.id == PVRRecordingsEvent::RecordingAdded)
```

Comment:
Are there many other events that can be triggered? Wondering if we should check whether we're interested in the events before looking up the EPG tags.

Reply:
Actually never mind, should have read the whole PR first...

Reply:
Currently there are no other events, but for the case we add more it is a good idea to precheck the event here. I will change that. Thanks.